test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -225,7 +235,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 				.toLowerCase() }`
 		);
 		const insertedBlockSelector = By.css(
-			`.block-editor-block-list__block.is-selected[aria-label*='Block: ${ name }']`
+			`.block-editor-block-list__block.${ selectedBlockConfirmClass }[aria-label*='${ name } Block']`
```

Comment:
What's going on here is that we're inserting a block that contains child blocks. When normally the entire block would then be selected, it's only the first child block in this case:

![pricing table](https://user-images.githubusercontent.com/96308/72727050-2474ae00-3bb4-11ea-811d-5cd4300ca112.gif)

Ideally, we'll find a way to make our selector accomodate for both cases :thinking: 

If we can't, I'd still suggest maybe handling the 'child block selected' case not through a generic-sounding `selectedBlockConfirmClass`, but maybe through a bool that determines whether we're dealing with a 'normal' block, or with one that has inner blocks, and set the relevant selector class to either `is-selected` or `has-child-selected`, depending on the value of that bool (so the code represents what's going on maybe a bit more intuitively).

Reply:
Yep, sounds like a good idea. I'll do a cleanup in the end to nail things down. :-)